packaging/linux/deb/package_binaries.sh

```diff
@@ -81,6 +81,14 @@ build_one_architecture() {
     > "$cron_file"
   chmod 755 "$cron_file"
 
+  # Whitelist for NativeMessaging
+  kbnm_bin="/usr/bin/kbnm"
+  kbnm_file="/etc/opt/chrome/native-messaging-hosts/io.keybase.kbnm"
```

Comment:
Definitely the kbnm_file one, fixed. The other one is a placeholder value for what gets written in the template.